test/src/e2e_vm_tests/test_programs/should_pass/language/logging/json_abi_oracle.json

```diff
@@ -1,79 +1,56 @@
 {
   "configurables": [],
+  "encoding": "1",
   "functions": [
     {
       "attributes": null,
       "inputs": [],
       "name": "main",
       "output": {
         "name": "",
-        "type": 14,
+        "type": 13,
         "typeArguments": null
       }
     }
   ],
   "loggedTypes": [
     {
-      "logId": 0,
+      "logId": 4579537983717831593,
       "loggedType": {
         "name": "",
-        "type": 7,
+        "type": 6,
         "typeArguments": []
       }
     },
     {
-      "logId": 1,
+      "logId": 16566583104751091389,
       "loggedType": {
         "name": "",
-        "type": 8,
+        "type": 7,
         "typeArguments": [
           {
             "name": "",
-            "type": 14,
+            "type": 13,
             "typeArguments": null
           }
         ]
       }
     },
     {
-      "logId": 2,
-      "loggedType": {
-        "name": "",
-        "type": 1,
-        "typeArguments": []
-      }
-    },
-    {
-      "logId": 3,
+      "logId": 5087777005172090899,
       "loggedType": {
         "name": "",
         "type": 1,
         "typeArguments": []
       }
     },
     {
-      "logId": 4,
+      "logId": 5555909392781521367,
```

Comment:
hmmm, technically js only support 52 bit numbers in json. So we either need to make this a string (which is breaking), or we have to truncate this hash further.

Reply:
Good catch! Making it a `string` should be easy enough.

cc @nedsalk